runtime/interpreter/primitivestatictype.go

```diff
@@ -200,6 +200,7 @@ const (
 	PrimitiveStaticTypePublicAccountKeys
 	PrimitiveStaticTypeAccountKey
 	PrimitiveStaticTypeAuthAccountInbox
+	PrimitiveStaticTypeCapabilityController
```

Comment:
Also make sure to handle this new type in relevant places, e.g. `PrimitiveStaticType.elementType()`, `PrimitiveStaticType.SemaType()`, etc.

Reply:
sorry, do you mean `elementSize`? can't find any other match with `rg -i 'primitivestatic.*element'`

Reply:
Yes, `elementSize`

Reply:
done